contract/src/main/kotlin/com/r3/corda/sdk/token/contracts/states/NonFungibleToken.kt

```diff
@@ -35,7 +35,13 @@ import net.corda.core.schemas.QueryableState
 open class NonFungibleToken<T : TokenType>(
         val token: IssuedTokenType<T>,
```

Comment:
Can we rename to `issuedTokenType`? Removes duplicated data.

```suggestion
override val issuedTokenType : IssuedTokenType<T>,
```